src/benchmarks/micro/libraries/System.IO/UnmanagedMemoryStreamTests.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+using System.Runtime.InteropServices;
+using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+
+namespace System.IO.Tests
+{
+    [BenchmarkCategory(Categories.Libraries)]
+    public unsafe class UnmanagedMemoryStreamTests
+    {
+        private const int Length = 10000;
+        private byte* buffer;
+
+        [GlobalSetup]
+        public void Setup()
+        {
+            buffer = (byte*)Marshal.AllocCoTaskMem(Length);
```

Comment:
Just wondering why you chose this over AllocHGlobal?

Reply:
No particular reason. Either of these two would work just fine for this test.